notebooks/book1/03/gauss_infer_2d.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>Please add meaningfull heading in 1st markdown cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>